src/Tizen.Multimedia.Vision/MediaVision/InferenceModelConfiguration.cs

```diff
@@ -0,0 +1,503 @@
+        public double StdValue
+        {
+            get
+            {
+                return GetDouble(_keyModelStdValue);
+            }
+            set
+            {
+                if (value <= 0)
+                {
+                    throw new ArgumentOutOfRangeException(nameof(value), value,
+                        $"Value can not be equal or less than 0");
+                }
+
+                Set(_keyModelStdValue, value);
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the inference model's backend engine.
+        /// </summary>
+        /// <remarks>The default backend type is <see cref="InferenceBackendType.OpenCV"/></remarks>
+        /// <exception cref="ArgumentException"><paramref name="value"/> is not valid.</exception>
+        /// <exception cref="NotSupportedException">The engine type is not supported.</exception>
+        /// <seealso cref="SupportedBackendType"/>
+        /// <since_tizen> 6 </since_tizen>
+        public InferenceBackendType Backend
+        {
+            get
+            {
+                return (InferenceBackendType)GetInt(_keyBackendType);
+            }
+            set
+            {
+                ValidationUtil.ValidateEnum(typeof(InferenceBackendType), value, nameof(Backend));
+
+                if (!SupportedBackendType.Contains(value))
+                {
+                    throw new NotSupportedException("Not supported engine type. " +
+                        "Please check supported engine using 'SupportedBackendType'.");
+                }
+
+                Set(_keyBackendType, (int)value);
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the inference model's target.
+        /// </summary>
+        /// <remarks>
+        /// The default target is <see cref="InferenceTargetType.CPU"/>.<br/>
+        /// Even if user set <see cref="InferenceTargetType.GPU"/> or <see cref="InferenceTargetType.Custom"/>,<br/>
+        /// if target doesn't support it, <see cref="InferenceTargetType.CPU"/> will be used internally.
+        /// </remarks>
+        /// <exception cref="ArgumentException"><paramref name="value"/> is not valid.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public InferenceTargetType Target
+        {
+            get
+            {
+                return (InferenceTargetType)GetInt(_keyTargetType);
+            }
+            set
+            {
+                ValidationUtil.ValidateEnum(typeof(InferenceTargetType), value, nameof(Target));
+
+                Set(_keyTargetType, (int)value);
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the size of inference model's tensor.
+        /// </summary>
+        /// <remarks>Both width and height of tensor should be greater than 0.</remarks>
+        /// <exception cref="ArgumentOutOfRangeException">The value is invalid.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public Size TensorSize
```

Comment:
Is it also valid on OpenCV backend?

Reply:
Yes, it's the general word as I heard from native owner.